dotnet/src/Planners/Planners.OpenAI/Stepwise/FunctionCallingStepwisePlanner.cs

```diff
@@ -136,7 +136,7 @@ public sealed class FunctionCallingStepwisePlanner
             // Look up function in kernel
             foreach (OpenAIFunctionToolCall functionResponse in functionResponses)
             {
-                if (clonedKernel.Plugins.TryGetFunctionAndArguments(functionResponse, out KernelFunction? pluginFunction, out KernelArguments? arguments))
+                if (clonedKernel.Plugins.TryGetOpenAIFunctionAndArguments(functionResponse, out KernelFunction? pluginFunction, out KernelArguments? arguments))
```

Comment:
We need to rename the method back to TryGetFunctionAndArguments. There's no reason to change it. It was done for Azure connector as well and that extension class will be dropped in favor of this one.

Reply:
fixed